test/runtests.jl

```diff
@@ -8,19 +8,41 @@ using MLJGLMInterface
 using GLM: coeftable
 import GLM
 import MLJTestInterface
+using Tables
 
 using  Distributions: Normal, Poisson, Uniform
 import StableRNGs
 using Tables
 
 expit(X) = 1 ./ (1 .+ exp.(-X))
 
-# TODO: Add more datasets to the following generic interface tests after #45 is merged
+# synthesize small data sets with mixed features:
+
+n = 100
+X_regression, y_regression = MLJBase.make_regression(n, 3)
+outlook = categorical(rand(["sunny", "overcast", "rainy"], n))
+temperature = categorical(
+    rand(["cold", "mild", "hot"], n),
+    ordered=true,
+    levels=["cold", "mild", "hot"],
+    )
+X = merge(
+    Tables.columntable(X_regression),
+    (; outlook, temperature),
+)
+y_binary = (temperature .== "mild" .|| outlook .== "sunny") |> categorical
```

Comment:
I don't really understand why this errors in julia 1.6. But probably writing it as `categorical(...)` should fix it?

Reply:
Will try that, thanks.